parlai/core/utils.py

```diff
@@ -248,7 +248,7 @@ def parse(txt, splitSentences=False):
         if useReplies == 'model' or (useReplies == 'label_else_model' and
                                      len(history['labels']) == 0):
             if reply != '':
-                history['dialog'].extend(parse(reply))
+                history['dialog'].extend(parse(reply, splitSentences))
```

Comment:
hmm, would this be affecting our kvmemnn too then or what?

Reply:
I'm not 100% certain. this actually caused an error here or I wouldn't have found it because it was returning:
`[ int, list, list, list, list, list ]`

this section should only be entered when the model reply is being used in the history. which, actually shouldn't be happening for memnn here either: it should use the labels. let me double check that.